apps/src/javalab/Neighborhood.js

```diff
@@ -34,5 +35,8 @@ export default class Neighborhood {
     controller.subtype.createDrawer(svg);
     controller.subtype.initWallMap();
     controller.initWithSvg(svg);
+
+    const slider = document.getElementById('slider');
+    this.speedSlider = new Slider(10, 35, 130, slider);
```

Comment:
since you already figured out how to use the speedSlider, want to add a comment todo with the equation we'll need to use to incorporate the slider?

Reply:
haha you added the comment right as I commented, never mind!